From 1ed7838c2a122c5c73992b9734bf27d5c1998d18 Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Tue, 24 Feb 2026 15:56:22 +0100
Subject: [PATCH] tests: py: Adjust payloads to changed userdata printing

JIRA: https://issues.redhat.com/browse/RHEL-128553
Upstream Status: nftables commit 471224bd15106fd94a182eaa9b794c484487f53f

commit 471224bd15106fd94a182eaa9b794c484487f53f
Author: Phil Sutter <phil@nwl.cc>
Date:   Thu Jan 29 14:49:17 2026 +0100

    tests: py: Adjust payloads to changed userdata printing

    libnftnl no longer prints userdata content but merely its size and a sum
    of all bytes.

    Signed-off-by: Phil Sutter <phil@nwl.cc>

Signed-off-by: Phil Sutter <psutter@redhat.com>
---
 tests/py/ip6/srh.t.payload | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/py/ip6/srh.t.payload b/tests/py/ip6/srh.t.payload
index 5c3031f..54fdd0e 100644
--- a/tests/py/ip6/srh.t.payload
+++ b/tests/py/ip6/srh.t.payload
@@ -11,7 +11,7 @@ ip6 test-ip6 input
 # srh last-entry { 0, 4-127, 255 }
 __set%d test-ip6 7 size 5
 __set%d test-ip6 0
-	element 00	element 01 flags 1	element 04	element 80 flags 1	element ff  userdata = { \x01\x04\x00\x00\x00\x01 }
+	element 00	element 01 flags 1	element 04	element 80 flags 1	element ff  userdata len 6 sum 0x6
 ip6 test-ip6 input
   [ exthdr load ipv6 1b @ 43 + 4 => reg 1 ]
   [ lookup reg 1 set __set%d ]
@@ -29,7 +29,7 @@ ip6 test-ip6 input
 # srh flags { 0, 4-127, 255 }
 __set%d test-ip6 7 size 5
 __set%d test-ip6 0
-	element 00	element 01 flags 1	element 04	element 80 flags 1	element ff  userdata = { \x01\x04\x00\x00\x00\x01 }
+	element 00	element 01 flags 1	element 04	element 80 flags 1	element ff  userdata len 6 sum 0x6
 ip6 test-ip6 input
   [ exthdr load ipv6 1b @ 43 + 5 => reg 1 ]
   [ lookup reg 1 set __set%d ]
@@ -47,7 +47,7 @@ ip6 test-ip6 input
 # srh tag { 0, 4-127, 0xffff }
 __set%d test-ip6 7 size 5
 __set%d test-ip6 0
-	element 0000	element 0001 flags 1	element 0004	element 0080 flags 1	element ffff  userdata = { \x01\x04\x00\x00\x00\x01 }
+	element 0000	element 0001 flags 1	element 0004	element 0080 flags 1	element ffff  userdata len 6 sum 0x6
 ip6 test-ip6 input
   [ exthdr load ipv6 2b @ 43 + 6 => reg 1 ]
   [ lookup reg 1 set __set%d ]
